.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: haskell
 
 ghc:
+ - 7.10
```

Comment:
Thanks for the pointer, Travis has 7.10 support but it should be written as "7.10", otherwise it will be parsed as 7.1, I'll fix that.